pkg/utils/chroot.go

```diff
@@ -149,12 +148,11 @@ func (c *Chroot) RunCallback(callback func() error) (err error) {
 	}()
 
 	// Store current root
-	oldRootF, err = c.config.Fs.Open("/")
+	rootPath, _ := c.config.Fs.RawPath("/")
```

Comment:
we no longer need to open the oldRoot directly to be able to chdir to it later, which is nice